### DISCUSSION TITLE
Why chat hacks should be returned to WCM

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1273] [(C) Suggestions]
#### Before reading my post, if you haven't seen #19 yet, please read the **entire thread**

# Chat hacks should be returned.

### Why?
Prodigy's chat is very limited. You can't even say all the item names, _ex. `Titan something something`( I forgor)_, let alone tell people to check out your house, ~edate~ nvm, advertise free memberships or hacks, talk like a normal person, tell funny jokes, or have civilized debates

### Swearing
I don't think that swearing should be stopped in prodigy chat. Either Prodigy fixes their problems, or they deal with their problems.
Why do we, PMGH, bother letting prodigy's rules control us?

### Links or ads
Also Prodigy's problem

### It's time we bring back chat hacks into WCM.
If you agree, please upvote.

**Comment by afkvido** [comment]
So.... is it true that this hack has been patched, and replaced with IDs that store messages in an array?

____
Open: #1243 

Closed because it "negatively impacted player experience": #19, 
First mention of chat hacks: #19,

Rejected because it's "not possible": #987, #890, #884, #685, #498, 
how is it not possible if we've literally had it before

Closed and ghosted on: #883, #174, 

Mentions of this hack being patched: #180,

**Reply by PatheticMustan** [reply]
#180 was not a "patch", it was just a name change.
Chat messages have ALWAYS been IDs in a message array.
While it is possible to add custom messages, the modified messages would only be shown to other hackers with the same message array.
Normal members would only see the original messages

**Reply by afkvido** [reply]
well.... having chat access even for just hackers is nice
